viite-UI/src/model/SelectedLinkProperty.js

```diff
@@ -178,6 +180,18 @@
         }
     };
 
+    var findUnknown = function(linkId, id) {
+      var permanent = !_.isUndefined(linkId) ? _.uniq(roadCollection.getTmpByLinkId([linkId]), _.isEqual) : (!_.isUndefined(id) ? _.uniq(roadCollection.getTmpById([id]), _.isEqual) : [] );
+      var temporary = !_.isUndefined(linkId) ? _.uniq(roadCollection.getTmpByLinkId([linkId]), _.isEqual) : (!_.isUndefined(id) ? _.uniq(roadCollection.getTmpById([id]), _.isEqual) : [] );
+      var permUnknown = _.reject(permanent, function(road) {
+        return road.roadLinkType !== RoadLinkType.UnknownRoadLinkType.value && road.anomaly !== Anomaly.None.value;
```

Comment:
if temporarary and permanent are equal then the "return road.getData().roadLinkType" line 187 and "return road.getData().roadLinkType" line 190 will lead to errors